File: DZ 6.1/types/Airplane.js
const { GraphQLObjectType, GraphQLInt } = require('graphql');

const AirplaneType = new GraphQLObjectType({
  name: 'Airplane',
  fields:() => (
    {
      airplane_id: { type: GraphQLInt },
      capacity: { type: GraphQLInt },
      type_id: { type: GraphQLInt },
      airline_id: { type: GraphQLInt }
    }
  ) 
});

module.exports = AirplaneType;

File: DZ 3.3+7.1/REST-API/REST-API/Models/Airplane.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace REST_API.Models
{
    public partial class Airplane
    {
        public Airplane()
        {
            Flights = new HashSet<Flight>();
        }

        public int AirplaneId { get; set; }
        public uint Capacity { get; set; }
        public int TypeId { get; set; }
        public int AirlineId { get; set; }

        public virtual AirplaneType Type { get; set; }
        public virtual ICollection<Flight> Flights { get; set; }
    }
}
